comp/core/workloadmeta/types.go

```diff
@@ -453,6 +527,47 @@ func (c Container) String(verbose bool) string {
 		}
 	}
 
+	if c.Owner != nil && c.Owner.Kind == KindECSTask && verbose {
```

Comment:
Should we add a test validating that the `String` function returns the correct value?